www/front_src/src/Resources/Graph/Performance/Graph/MetricsTooltip.tsx

```diff
@@ -23,7 +24,7 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     flexDirection: 'column',
   },
-  dateTime: {
+  emphasized: {
```

Comment:
Why this term ? "emphasized" ?

Reply:
You can check emphasis here : https://material.io/design/color/text-legibility.html#text-backgrounds
in french, It's like "mettre en valeur"

Reply:
Yes ! ok thanks :)

Reply:
it means that the item is presented with more importance